qi-test/tests/definitions.rkt

```diff
@@ -81,7 +81,9 @@
     (check-false ((π args (apply > _)) 1 2 3) "apply with packed args")
     (check-true ((π args (apply > _)) 3 2 1) "apply with packed args")
     (check-equal? ((π a* _) 1 2 3 4) '(1 2 3 4))
-    (check-equal? ((π (a . a*) list) 1 2 3 4) '(1 (2 3 4))))
+    (check-equal? ((π (a . a*) list) 1 2 3 4) '(1 (2 3 4)))
+    (check-equal? ((π (a #:b b . a*) list) 1 2 3 4 #:b 'any) '(1 (2 3 4)))
+    (check-equal? ((π (a #:b b c . a*) list) 1 2 3 4 #:b 'any) '(1 2 (3 4))))
```

Comment:
Also need a test here to use the bound value from the keyword, e.g.:
`(check-equal? ((flow-lambda (a b #:c c) (~> + (* c))) 2 3 #:c 10) 50)`